client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1831,6 +1832,7 @@
   "select-report": "Select Template",
   "select-store": "Select store",
   "sensors": "Sensors",
+  "server-error.DemographicIndicatorHasNoName": "Demographic indicator has no name",
```

Comment:
camelCase in key not ideal here, but seemed wasteful to convert server error to snake-case just so it looks prettier in here.